client_test.go

```diff
@@ -155,8 +155,6 @@ func newTestClient(t *testing.T, dbPool *pgxpool.Pool, config *Config) *Client[p
 	client, err := NewClient(riverpgxv5.New(dbPool), config)
 	require.NoError(t, err)
 
-	client.testSignals.Init()
```

Comment:
I was having trouble with my stress test where the test signal channels were filling up, and unfortunately because init'ing test signals also inits test signals for all maintenance services, it means there's no way to deinit them again. The only decent answer is to not init them on every single client initialization, and move inits into test cases that need them.